src/modules/navigator/mission_feasibility_checker.cpp

```diff
@@ -87,6 +87,10 @@ MissionFeasibilityChecker::checkMissionFeasible(const mission_s &mission,
 	    || _navigator->get_vstatus()->is_vtol) {
 		failed = failed || !checkRotarywing(mission, home_alt);
 
+		if (_navigator->get_vstatus()->is_vtol) {
```

Comment:
This might read a bit easier with vtol as a separate case.

if vtol
elseif fw
elseif mc

Reply:
That's true, thanks.